frontend/routes/components/dialogContainer/components/simpleErrorDialog/simpleErrorDialog.component.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
```

Comment:
Do we really need that template? Isn't default error dialog enough?

Reply:
@dleszcz

Reply:
@simplypixi ok, I used ErrorDialog instead of this new template